components/clarity-vscode/README.md

```diff
@@ -1,50 +1,69 @@
 
-The debugger included in this extension allows to run Clarity code line by line which is a powerful tool to understand what precisely happens. Read this [blog post](https://www.hiro.so/blog/how-to-debug-your-smart-contracts-with-clarinet) to know more about Clarity debugging.
+You can install the latest release of the plugin from the [marketplace](https://marketplace.visualstudio.com/items?itemName=hirosystems.clarity-lsp).
+## Features
 
-In order to use the Debugger (DAP), this extension relies on a local installation of Clarinet. To install Clarinet, please follow the instructions [here](https://github.com/hirosystems/clarinet#installation).
+The features described below are available when you install the Clarity for Visual Studio plugin.
 
----
-## Contributes
+### Auto Complete Native Functions
 
-### Run the extension locally
+This feature enables you to start typing a function name, and then have the editor automatically suggest auto-completion with the documentation related to the suggestion.
 
-You'll need to have Rust, Node.js and NPM installed.
+When you select a function, the extension adds the necessary parentheses around it and puts placeholders in the arguments of the function.
 
-From the `./components/clarity-vscode`, run `npm install` to install the dependencies and `npm run dev` to start the extension in a Chromium instance.
+![autocomplete gif](images/autocomplete.gif)
 
-### Structure
+### Check Contract on Save and Display Errors Inline
 
-The LSP has two main parts: the client and the server.
-This two part will run differents environments:
-- VSCode Web: in a WebWorkers
-- VSCode Desktop: in a Node.js environments
+When a contract is opened or saved, the extension will notify you if errors are found (syntax, unknown keyword, etc), or warnings (such as unsafe code). This helps you to ensure that you write safe and clean code.
 
-The LSP (`./components/clarity-lsp`), written in Rust, is built with wasm-pack for both these environments and will be load accordingly, with `fetch()` in the browser and `fs.readFileSync()` in Node.js.
+![display errors gif](images/errors.gif)
 
-```
-./components/clarity-vscode
-├── package.json // The extension manifest.
-├── client
-│   └── src
-│       ├── clientBrowser.ts
-│       ├── clientNode.ts
-│       └── common.ts
-└── server
-    └── src
-        ├── serverBrowser.ts
-        ├── serverNode.ts
-        └── common.ts
-```
+### VS-Code Support
+
+Visual Studio Code is natively supported, but this extension provides additional support for [vscode.dev](https://vscode.dev/), [github.dev](https://github.dev/github/dev) and similar tools.
+
+For more information, please see the [Clarinet GitHub VS Code documentation page](https://github.com/hirosystems/clarinet/blob/develop/components/clarity-vscode/README.md)
+
+### Auto Complete User-Defined Functions
+
+This feature is similar to the auto-complete feature, but does not provide documentation for functions when writing a contract.
```

Comment:
I don't think we should mention "it doesn't provide documentation" for user defined function, since user defined function don't really  have documentation in the first place.

Reply:
Maybe we can remove it and only change the first feature ("Autocomplete native functions"), with "Autocomplete functions" (native and user defined)

Reply:
fixed